examples/developer-tools/src/index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Blockly Developer Tools</title>
+  </head>
+  <body>
+    <div id="blockFactoryContainer">
+      <div id="mainWorkspace"></div>
```

Comment:
Kebab case for id attributes: https://google.github.io/styleguide/htmlcssguide.html#id_Attributes